packages/vuetify/src/components/VSelect/VSelectList.ts

```diff
@@ -206,12 +207,12 @@ export default mixins(Colorable, Themeable).extend({
         ? this.$createElement(VListItem, tile, scopedSlot)
         : scopedSlot
     },
-    genTileContent (item: any): VNode {
+    genTileContent (item: any, index = 0 as string | number): VNode {
```

Comment:
```suggestion
    genTileContent (item: any, index = 0 as string): VNode {
```

Reply:
0 isn't a string

Reply:
```suggestion
    genTileContent (item: any, index: string | number = 0): VNode {
```

Reply:
removed the wrong type...

Reply:
i've edited my comment (just removed the wrong type before), string is not necessary here

Reply:
neither is `as number`